components/ui/BannerCarousel.tsx

```diff
@@ -45,184 +53,124 @@ export interface Props {
-          height={600}
+          height={441}
         />
         <Source
           media="(min-width: 768px)"
           fetchPriority={lcp ? "high" : "auto"}
           src={desktop}
           width={1440}
-          height={600}
+          height={441}
         />
         <img
-          class="object-cover w-full h-full"
+          class="object-cover w-full"
           loading={lcp ? "eager" : "lazy"}
           src={desktop}
           alt={alt}
         />
       </Picture>
-      {action && (
-        <div class="absolute h-min top-0 bottom-0 m-auto left-0 right-0 sm:right-auto sm:left-[12%] max-h-min max-w-[400px] flex flex-col gap-4 p-4 rounded glass">
-          <span class="text-6xl font-medium text-base-100">
-            {action.title}
-          </span>
-          <span class="font-medium text-xl text-base-100">
-            {action.subTitle}
-          </span>
-          <Button class="glass">{action.label}</Button>
-        </div>
-      )}
+
+      <BannerActionLayout action={action} />
     </a>
   );
 }
 
-function Dots({ images, interval = 0 }: Props) {
-  return (
-    <>
-      <style
-        dangerouslySetInnerHTML={{
-          __html: `
-          @property --dot-progress {
-            syntax: '<percentage>';
-            inherits: false;
-            initial-value: 0%;
-          }
-          `,
-        }}
-      />
-      <ul class="carousel justify-center col-span-full gap-4 z-10 row-start-4">
-        {images?.map((_, index) => (
-          <li class="carousel-item">
-            <Slider.Dot index={index}>
-              <div class="py-5">
-                <div
-                  class="w-16 sm:w-20 h-0.5 rounded group-disabled:animate-progress bg-gradient-to-r from-base-100 from-[length:var(--dot-progress)] to-[rgba(255,255,255,0.4)] to-[length:var(--dot-progress)]"
-                  style={{ animationDuration: `${interval}s` }}
-                />
-              </div>
-            </Slider.Dot>
-          </li>
-        ))}
-      </ul>
-    </>
-  );
-}
+export const BannerActionLayout = ({ action }: { action?: ActionProps }) => {
+  if (action) {
+    return (
+      <div class=" absolute lg:max-w-3xl p-4 lg:p-0  md:top-[-20px]  max-[900px]:top-[-20px]  h-min top-0 bottom-0 m-auto left-0 right-0 text-center  max-h-min  flex flex-col items-center">
+        {action?.title && (
+          <span
+            class="font-medium text-3xl text-white mb-6 md:mb-2 max-[900px]:mb-2"
+            dangerouslySetInnerHTML={{
+              __html: action.title ?? "",
+            }}
+          ></span>
+        )}
+        {action?.subTitle && (
+          <span
+            class="font-normal  text-base text-white mb-9 md:mb-5 lg:mb-9 "
+            dangerouslySetInnerHTML={{
+              __html: action.subTitle ?? "",
+            }}
+          ></span>
+        )}
+        {action?.label && (
+          <Button class=" w-[calc(60%)] md:w-[calc(50%-36px)] text-blueNew bg-white h-9 px-2 py-3 text-center ">
```

Comment:
1. calcs desnecessarios
2. use as cores semanticas que estao no Theme: https://admin.deco.cx/sites/persono/blocks/Default%20Theme
3. use os componentes do daisy UI: https://daisyui.com/components/button/
```suggestion
          <Button class="w-3/5 md:w-1/2 btn-primary btn-outline">
```